lib/metanorma/cli/commands/site.rb

```diff
@@ -13,9 +14,14 @@ class Site < Thor
           default: Pathname.new(Dir.pwd).join("site").to_s,
           desc: "Output directory for generated site",
         )
+        option :"agree-to-terms", type: :boolean, desc: "Agree / Disagree with all third-party licensing terms "\
+                                                        "presented (WARNING: do know what you are agreeing with!)"
+        option :"no-install-fonts", type: :boolean, desc: "Skip the font installation process"
+        option :"continue-without-fonts", type: :boolean, desc: "Continue processing even when fonts are missing"
```

Comment:
For the options name doesn't thor automatically handle this symbol to string conversion? `agree_to_terms` -> `agree-to-terms`, any specific reason why we are doing it by ourself?

Reply:
Agree -- didn't know Thor automatically handles them. @CAMOBAP can you help fix this?

Reply:
Yep we have a separate ticket for this https://github.com/metanorma/metanorma-cli/issues/218 and related PRs is ready for review already